NEWS.md

```diff
@@ -1,3 +1,7 @@
+# tic 0.2.13.9002
+
+- The `openssl` package moved from 'Suggests' to 'Imports' because `get_public_key()` and `get_private_key()` were moved from package `travis` to here.
```

Comment:
I can handle NEWS entries and version bumps.

Reply:
Ok, reset to `master`.